src/mixins/render-vega-lite/Definitions/Encoding/Scale/Continuous/ContinuousScale.js

```diff
@@ -127,10 +135,32 @@ export default class ContinuousScale extends ScaleDefinitionObject {
       ({ formula_xform_objs, vega_xform_outputs }) => {
         if (vega_xform_outputs.length === 0) {
           assert(formula_xform_objs.length >= 2)
-          vega_xform_outputs.push(
-            formula_xform_objs[formula_xform_objs.length - 2].as,
-            formula_xform_objs[formula_xform_objs.length - 1].as
-          )
+          const start_range_formula =
+            formula_xform_objs[formula_xform_objs.length - 2]
+          const end_range_formula =
+            formula_xform_objs[formula_xform_objs.length - 1]
+          vega_xform_outputs.push(start_range_formula.as)
+          if (num_subdivisions > 0) {
+            const diff_var_name = `${field_output}_extents_diff`
+            formula_xform_objs.push({
+              type: "formula",
+              expr: `(${end_range_formula.as} - ${
+                start_range_formula.as
+              }) / ${num_subdivisions + 1}.0`,
+              as: diff_var_name
+            })
+
+            for (let i = 1; i <= num_subdivisions; ++i) {
+              const var_name = `${field_output}_extents_stop_${i}`
+              formula_xform_objs.push({
```

Comment:
now compute each individual subdivision

NOTE: this could be more easily handled with some sort of array functions that linearly auto-fills subdivisions between a range. But since that's not currently possible on the BE, we need to compute each stop manually. Fortunately we can do that.